rts/Game/UI/KeyBindings.cpp

```diff
@@ -305,19 +305,15 @@ void CKeyBindings::Kill()
 
 
 /******************************************************************************/
-const CKeyBindings::ActionList& CKeyBindings::GetActionList(const std::string& matcher) const
+const CKeyBindings::ActionList& CKeyBindings::GetActionList() const
 {
 	static ActionList merged; //FIXME switch to thread_local (?)
```

Comment:
This problem can be consolidated into one declaration instead, I copied the previous method. Notice also the ported FIXME, which I'm not sure what it refers to.

Reply:
I don't think the FIXME is a concern at the moment. It isn't clear how thread local storage would be helpful at the moment.